<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPage</class>
 <widget class="QTabWidget" name="SearchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="overviewTab">
   <attribute name="title">
    <string>Всё</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="overviewScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="overviewContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>250</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="trackTab">
   <attribute name="title">
    <string>Треки</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="trackLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Треков: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tracksTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="albumTab">
   <attribute name="title">
    <string>Альбомы</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="albumLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Альбомов: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="albumScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="albumContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>229</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QGridLayout" name="albumGrid_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="playlistTab">
   <attribute name="title">
    <string>Плейлисты</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="playlistLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Плейлистов: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="playlistScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="playlistContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>229</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QGridLayout" name="playlistGrid"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="artistTab">
   <attribute name="title">
    <string>Исполнители</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="artistLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Исполнителей: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="artistScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="artistContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>229</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QGridLayout" name="artistGrid"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="userTab">
   <attribute name="title">
    <string>Профили</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="userLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Профилей:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="userScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="userContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>229</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QGridLayout" name="userGrid"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="channelTab">
   <attribute name="title">
    <string>Каналы</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="channelLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Каналов: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="channelScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="channelContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>229</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QGridLayout" name="channelGrid"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="podcastTab">
   <attribute name="title">
    <string>Подкасты</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QLabel" name="podcastLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Подкастов: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="podcastScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="podcastContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>229</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QGridLayout" name="podcastGrid"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="episodeTab">
   <attribute name="title">
    <string>Выпуски</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <widget class="QLabel" name="episodeLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выпусков:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="episodeScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="episodeContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>229</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <layout class="QGridLayout" name="episodeGrid"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="mixTab">
   <attribute name="title">
    <string>Миксы</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_17">
    <item>
     <widget class="QLabel" name="mixLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Миксов: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="mixScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="mixContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>229</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <layout class="QGridLayout" name="mixGrid"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
